smartypants.go

```diff
@@ -398,13 +400,14 @@ func NewSmartypantsRenderer(flags Extensions) *SPRenderer {
 	return &r
 }
 
-func (sr *SPRenderer) Process(text []byte) []byte {
+// Process is the entry point of the Smartypants renderer.
+func (smrt *SPRenderer) Process(text []byte) []byte {
```

Comment:
I don't have anything against it, but why `sr` -> `smrt` change? What does `smrt` stand for? Was this a golint issue? Just wondering.